packages/grafana-schema/src/common/variables.cue

```diff
@@ -0,0 +1,36 @@
+package common
+
+// Optional formats for the template variable replace functions
+VariableFormatID: 
+    // Values are lucene escaped and multi-valued variables generate an OR expression
```

Comment:
@sdboyer -- thoughts on how to get comments on individual elements passed through?

Reply:
this one is tricky, because AFAIK CUE's parser doesn't treat branches of a disjunction as discrete AST nodes to which comments can be attached.

If that is indeed the case, it means we need to come up with some other way of declaring enums in order to be able to attach comments to them.

simplest would probably be declaring each element as a standalone field, allowing us to attach comments. I'd need to explore some more to come up with a suggested pattern.

Reply:
that makes sense -- I kinda figured :)

Anything *bad* about using this style for now?

Reply:
go for it imo